lib/install/bootstrap/install.rb

```diff
@@ -0,0 +1,8 @@
+say "Install Bootstrap 5 with Popperjs/core"
+copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap_js_files.js"
```

Comment:
Is it just me or does the filename `bootstrap_js_files.js` feel weird? 

I would have expected something like `bootstrap.js`.

```suggestion
copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap.js"
```

Reply:
@marcoroth agree! I renamed and pushed the changes to the current fork.